gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,17 +1,7 @@
-#
```

Comment:
We probably need to keep this header, but since this is auto-generated, may be it is not needed

Reply:
I'm spoiled on SDK where spotless fixes all the license header (and newline) issues for me.  I'll put it back in even if it's not needed, although the next time someone runs `gradle wrapper` it'll delete it again!